hugegraph-dist/pom.xml

```diff
@@ -178,22 +178,9 @@
                             <goal>run</goal>
                         </goals>
                         <configuration>
-                            <tasks>
-                                <echo file="${top.level.dir}/dist.sh">
-                                    tar -zcvf \
-                                    ${top.level.dir}/${final.name}.tar.gz \
-                                    ${final.name} || exit 1
-                                    rm -f ${top.level.dir}/dist.sh
-                                    echo
-                                    echo "HugeGraph dist tar.gz available at:
-                                    ${top.level.dir}/${final.name}.tar.gz"
-                                    echo
-                                </echo>
-                                <exec executable="${shell-executable}"
-                                      dir="${top.level.dir}" failonerror="true">
-                                    <arg line="./dist.sh"/>
-                                </exec>
-                            </tasks>
+                            <target>
+                                <tar destfile="${top.level.dir}/${final.name}.tar.gz" basedir="${top.level.dir}"  includes="${final.name}/*" compression="gzip"/>
```

Comment:
can wrap line for every property?

Reply:
like this
![image](https://user-images.githubusercontent.com/13429553/204692364-0ffd59d8-6477-4652-b501-7f4dd23d1983.png)

Reply:
yes